homeassistant/components/weather/__init__.py

```diff
@@ -482,7 +482,7 @@ def _default_wind_speed_unit(self) -> str:
         Should not be set by integrations.
         """
         return (
-            SPEED_KILOMETERS_PER_HOUR
+            SPEED_METERS_PER_SECOND
```

Comment:
This is changing **all** weather integrations of Home Assistant. I'm pretty sure that is not correct.

Reply:
You might be write but I have been following the code from my issue (https://github.com/home-assistant/frontend/issues/13161) and this is where is see it being assigned a different unit. If I am wrong I can cancel it.

Reply:
This change is not adjusting it for met_eireann, but for ALL weather integrations. 
Are you saying all weather integrations are incorrect?

Reply:
The met_eireann weather platform already sets the native wind speed unit to meters per second.

https://github.com/home-assistant/core/blob/0006629ca23455eb3d16ad28774660a3e84a7a45/homeassistant/components/met_eireann/weather.py#L61

The user can convert the measurement to another supported unit if they like:
https://developers.home-assistant.io/docs/core/entity/weather#unit-conversion

Reply:
Yes Met Eireann is in M/S (I have checked the raw data via my terminal). But some code somewhere is converting to km/hr but appending m/s as the units. When the above Unit conversion is selected it converts a second time. For example it was saying my wind speed in km/hr was hurricane speed. It is multiplying it by 3.6 every time.

Reply:
Please open an issue with this information so we can track it and collect information. Thanks!

Reply:
At this very moment the raw (http://metwdb-openaccess.ichec.ie/metno-wdb2ts/locationforecast?lat=54.7210798611;long=-8.7237392806) is 1.5 m/s but in HA its 5.4 m/s which is 3.6 more (m/s to km/hr) :) but the units are m/s

Reply:
Please open an issue with all this information so we can continue troubleshooting there. Thanks!

Reply:
Here is the issue. https://github.com/home-assistant/frontend/issues/13161